msmbuilder/feature_selection/featureselector.py

```diff
@@ -0,0 +1,60 @@
+# Author: Carlos Xavier Hernandez <cxh@stanford.edu>
+# Contributors:
+# Copyright (c) 2016, Stanford University and the Authors
+# All rights reserved.
+import numpy as np
+
+from .base import MultiSequenceFeatureSelectionMixin
+
+
+class FeatureSelector(MultiSequenceFeatureSelectionMixin):
```

Comment:
We should add a warning that you need to normalize the dataset afterwards. Mixing and matching featurizers with differing means and variances is gonna cause a lot of issues later on. Alternate option is to add a sklearn normalizer as a attribute to the object and it get auto called when fitting.